nornir_nautobot/plugins/tasks/dispatcher/__init__.py

```diff
@@ -12,6 +12,7 @@
 _DEFAULT_DRIVERS_MAPPING = {
     "default": "nornir_nautobot.plugins.tasks.dispatcher.default.NautobotNornirDriver",
     "default_netmiko": "nornir_nautobot.plugins.tasks.dispatcher.default.NetmikoNautobotNornirDriver",
+    "cisco_asa": "nornir_nautobot.plugins.tasks.dispatcher.default.NetmikoNautobotNornirDriver",
```

Comment:
Can we follow the same pattern as the others? I would prefer to have this in it's own file, even if nothing is being done, so that in the future there is no backwards compat issues

Reply:
You bet.  Will update.